### PR TITLE
Added softmax

### PR DESCRIPTION
**Pull request by aryaman4152**
Added softmax after fully connected layer in Resnet

### CODE REVIEW

**Comment by aryaman4152** [comment]
Don't need softmax in pytorch
